unstructured_ingest/v2/processes/connectors/astradb.py

```diff
@@ -325,29 +325,6 @@ def conform_dict(self, element_dict: dict, file_data: FileData) -> dict:
             "metadata": element_dict,
         }
 
```

Comment:
Just curious. Never seen this. Why does line 322 have a dollar sign before vector?

Reply:
I guess its just how the sdk wants it... :)